lm/reverse_arpa.py

```diff
@@ -22,163 +33,107 @@ def run(self):
         # Copyright 2012 Mirko Hannemann BUT, mirko.hannemann@gmail.com
         # Copied from Kaldi: http://sourceforge.net/p/kaldi/code/1640/tree//trunk/egs/wsj/s5/utils/reverse_arpa.py
         # Adapted for python 3
```

Comment:
the copyright notice should be removed here and maybe added to the `lib/lm.py`
```suggestion
```

Reply:
I'd put it next to the reverse code. `lib/lm.py` only contains I/O code.

Reply:
is it fine then?